config/opal_mca.m4

```diff
@@ -149,7 +149,7 @@ AC_DEFUN([OPAL_MCA],[
                 AC_MSG_ERROR([*** The enable-mca-direct flag requires a
 *** list of type-component pairs.  Invalid input detected.])
             else
-                AS_VAR_COPY([AS_TR_SH([DIRECT_$type])], [AS_TR_SH([$comp])])
+                AS_VAR_SET([AS_TR_SH([DIRECT_$type])], [AS_TR_SH([$comp])])
```

Comment:
I believe `AS_VAR_SET` Is correct here too, because, simplifying, it would be either `AS_VAR_COPY([DIRECT_foo], [comp])` or `AS_VAR_SET([DIRECT_foo], [$comp])` but not `AS_VAR_COPY([DIRECT_foo], [$comp])`. But I didn't test this path.